### PR TITLE
Added BoundedSet assignment and tests as per request.

### PR DESCRIPTION
**Pull request by ZakBibi**

### CODE REVIEW

File: CodingPractice/OOPPractice/assignments/BoundedSet.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+    def __init__(self, limit_1, limit_2):
+        self.limit_1 = limit_1
+        self.limit_2 = limit_2
+        self.inner_set = set()
+
+    def put(self, element):
+        if element >= self.limit_1 and element <= self.limit_2:
+            self.inner_set.add(element)
+        else:
```
Comment (BasilBibi) [inline CodingPractice/OOPPractice/assignments/BoundedSet.py]: No need to print that the element falls outside of the bounds.

File: CodingPractice/OOPPractice/tests/test_BoundedSet.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from CodingPractice.OOPPractice.assignments.BoundedSet import *
+
+
```
Comment (BasilBibi) [inline CodingPractice/OOPPractice/tests/test_BoundedSet.py:4]: Your tests should make sure that element values either side of the lower and upper bound are handled correctly.
To use your example BoundedSet(1,10) I would expect to see you checking that the following are correctly handled : 0, 1, 10  and 11.
You can check this quickly with and assert using an expected value of  set(1,2,3,4,5,6,7,8,9, 10)

File: CodingPractice/OOPPractice/tests/test_BoundedSet.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+class TestBoundedSet(unittest.TestCase):
+
+    def test_set_boundries(self):
+        a = BoundedSet(1, 10)
+        a.put(3)
```
Comment (BasilBibi) [inline CodingPractice/OOPPractice/tests/test_BoundedSet.py:9]: Why is this a.get() here is is doing nothing?

File: CodingPractice/OOPPractice/tests/test_BoundedSet.py (hunk 2 above)
Comment (BasilBibi) [reply CodingPractice/OOPPractice/tests/test_BoundedSet.py:4]: You need to test that the boundary conditions are satisfied 
e.g. 0 not in a.get() 
11  not in a.get()

File: CodingPractice/OOPPractice/tests/test_BoundedSet.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        a.get()
+        self.assertEqual({3}, a.get())
+
+    def test_boundries_fail(self):
+        b = BoundedSet(1, 10)
+        b.put(11)
```
Comment (BasilBibi) [inline CodingPractice/OOPPractice/tests/test_BoundedSet.py]: This makes no sense as the print statement is not returned.

File: CodingPractice/OOPPractice/assignments/BoundedSet.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class BoundedSet:
+
```
Comment (BasilBibi) [inline CodingPractice/OOPPractice/assignments/BoundedSet.py:2]: These variable names are not very descriptive - how about upper_bound and lower_bond ?

**Review by BasilBibi — CHANGES_REQUESTED** [review]
